### ISSUE TITLE
place picker just show only few seconds

### ISSUE THREAD

**Issue by akiladevinda** [q6112345/react-native-google-place-picker#44]
When start intent google place it show up google search place on 1-2 second it will be auto dismiss. 

This automatically runs the didCancel method

**Comment by pankti1210** [comment]
must be a problem with an API key.. 
i had the same issue but after regenerating the API key, it's get resolved.

**Comment by akiladevinda** [comment]
> must be a problem with an API key..
> i had the same issue but after regenerating the API key, it's get resolved.

I did it ... but still i have this issue

**Comment by pankti1210** [comment]
@akiladevinda got any solution ?

**Comment by akiladevinda** [comment]
@pankti1210 No.. I have generated new API key. But the issue is till have. I would like to know weather you have Google Map Billing Account ? for enabling places sdk ? 

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/20571997/59417849-9e0a1980-8de5-11e9-9b06-545e95ca23b8.gif)

**Comment by akiladevinda** [comment]
This is the sample video of my problem 
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/20571997/59417752-6ef3a800-8de5-11e9-9964-eb6004bf2ad7.gif)
